application/listeners/listeners.go

```diff
@@ -101,7 +103,19 @@ func userCreated(e events.Event) {
 		return
 	}
 
+	user, ok := e.Payload["user"].(*models.User)
+	if !ok {
+		domain.Logger.Printf("%s Failed to get User from event payload for notification. Event message: %s",
+			domain.GetCurrentTime(), e.Message)
```

Comment:
I haven't been adding the time to log messages, do you think I should? FWIW, CloudWatch logs the time too. Is that redundant?

Reply:
okay, yeah I'd say it's redundant and we can remove it. perhaps we had it there for stdout to our console, but that we'd be watching realtime during dev/test anyway